sdk/resourcemanager/azure-resourcemanager-authorization/src/main/java/com/azure/resourcemanager/authorization/models/BuiltInRole.java

```diff
@@ -211,6 +211,29 @@
     public static final BuiltInRole KEY_VAULT_CRYPTO_SERVICE_ENCRYPTION_USER =
         BuiltInRole.fromString("Key Vault Crypto Service Encryption User");
 
+    // AKS related roles
+    /** Lets you manage all resources in the cluster. */
+    public static final BuiltInRole AZURE_KUBERNETES_SERVICE_RBAC_CLUSTER_ADMIN =
```

Comment:
Where are those built-in roles used?

Reply:
They are for user. Basically this class is for user.

Reply:
Update javadoc for detail https://github.com/Azure/azure-sdk-for-java/pull/27729/commits/03016de9de6ec16edbba75f2b80d5d8f0888af2c